cms/djangoapps/contentstore/tests/test_import.py

```diff
@@ -283,3 +283,54 @@ def _verify_split_test_import(self, target_course_name, source_course_name, spli
         }
 
         self.assertEqual(remapped_verticals, split_test_module.group_id_to_child)
+
+    @ddt.data(ModuleStoreEnum.Type.mongo, ModuleStoreEnum.Type.split)
+    def test_video_components_present_while_import(self, store):
```

Comment:
@nasthagiri i investigate it more it seems the problem occurs when ```target_id``` is same while re-importing , because when ```target_id``` is different its just like importing the course into new instance , is this correct behaviour ?